php/commands/core.php

```diff
@@ -742,39 +742,59 @@ private static function get_clean_basedomain() {
 	 * @when before_wp_load
 	 */
 	public function version( $args = array(), $assoc_args = array() ) {
-		$versions_path = ABSPATH . 'wp-includes/version.php';
-
-		if ( !is_readable( $versions_path ) ) {
-			WP_CLI::error(
-				"This does not seem to be a WordPress install.\n" .
-				"Pass --path=`path/to/wordpress` or run `wp core download`." );
-		}
-
-		include $versions_path;
+		$version = self::get_version();
 
 		// @codingStandardsIgnoreStart
 		if ( \WP_CLI\Utils\get_flag_value( $assoc_args, 'extra' ) ) {
-			if ( preg_match( '/(\d)(\d+)-/', $tinymce_version, $match ) ) {
+			if ( preg_match( '/(\d)(\d+)-/', $version['tinymce'], $match ) ) {
 				$human_readable_tiny_mce = $match[1] . '.' . $match[2];
 			} else {
 				$human_readable_tiny_mce = '';
 			}
 
 			echo \WP_CLI\Utils\mustache_render( 'versions.mustache', array(
-				'wp-version' => $wp_version,
-				'db-version' => $wp_db_version,
+				'wp-version' => $version['wp'],
+				'db-version' => $version['db'],
 				'mce-version' => ( $human_readable_tiny_mce ?
-					"$human_readable_tiny_mce ($tinymce_version)"
-					: $tinymce_version
+					"$human_readable_tiny_mce ({$version['tinymce']})"
+					: $version['tinymce']
 				)
 			) );
 		} else {
-			WP_CLI::line( $wp_version );
+			WP_CLI::line( $version['wp'] );
 		}
 		// @codingStandardsIgnoreEnd
 	}
 
 	/**
+	 * Get version information from `wp-includes/version.php`.
+	 *
+	 * @return array {
+	 *     @type string $wp The WordPress version.
+	 *     @type int $db The WordPress DB revision.
+	 *     @type string $tinymce The TinyMCE version.
+	 * }
+	 */
+	private static function get_version() {
+		$versions_path = ABSPATH . 'wp-includes/version.php';
+
+		if ( !is_readable( $versions_path ) ) {
+			WP_CLI::error(
+				"This does not seem to be a WordPress install.\n" .
+				"Pass --path=`path/to/wordpress` or run `wp core download`." );
+		}
+
+		include $versions_path;
+
+		return array(
+			'wp' => $wp_version,
+			'db' => $wp_db_version,
+			'tinymce' => $tinymce_version,
+			'local_package' => $wp_local_package
```

Comment:
> Does WordPress redefine this global somewhere? (I've done a quick search and it doesn't seem to be the case.)

I was under the assumption it could be repopulated by the database. But, on second thought, I don't see why that should impact verifying checksums.

@szepeviktor Thoughts?